test/TestServerProjects/azure-parameter-grouping/Generated/Models/Error.Serialization.cs

```diff
@@ -5,39 +5,9 @@
 
 #nullable disable
 
-using System.Text.Json;
-using Azure.Core;
-
 namespace azure_parameter_grouping.Models
 {
     public partial class Error
     {
-        internal static Error DeserializeError(JsonElement element)
```

Comment:
So, I'd have to change it so that, when exceptions are generated, we don't generate this file, right? Because, we aren't using the deserializer.

Reply:
Hm, let's generate the deserializer instead.

Reply:
This is a bug. So, I'll fix the bug.

Reply:
Fixed.